src/components/explore/AggregationsOverTimeGraph.svelte

```diff
@@ -202,13 +205,17 @@
       yScaleType === 'scalePoint'
     ) {
       visiblePercentiles.forEach((p) => {
-        yData = yData.concat([...data.map((arr) => arr[percentileName][p])]);
+        yData = yData.concat([
+          ...data.map((arr) => arr[percentileName] && arr[percentileName][p]),
+        ]);
       });
       // use transformed data for Android metrics
-      if (data[0].transformedPercentiles) {
+      if (data[data.length - 1].transformedPercentiles) {
```

Comment:
we use lengh - 1, instead of 0, because non-normalized data is added later. so the first entry of data array usually doesn't have non-normalized data available